content/docs/concepts/enterprise_users.md

```diff
@@ -0,0 +1,109 @@
++++
+title = "NSM Concepts"
+description = "for Enterprise Users"
+short = "Enterprise Users"
+weight = 1
+date = "2021-06-21"
++++
+## The Problem
+
+### Background
+Historically, workloads have been run in some sort of Runtime Domain:
+
+![K8s Runtime Domain](/img/concepts/enterprise/3x1/k8s_runtime_domain.svg)
+![VM Runtime Domain](/img/concepts/enterprise/3x1/vm_runtime_domain.svg)
+![DC Runtime Domain](/img/concepts/enterprise/3x1/dc_runtime_domain.svg)
+
+A Runtime Domain is a system on which workloads are run.  It's fundamentally a compute domain.
+
+Each of those Runtime Domains have brought along exactly one Connectivity Domain:
+
+![K8s Connectivity Domain](/img/concepts/enterprise/3x1/k8s_connectivity_domain.svg)
+![VM Connectivity Domain](/img/concepts/enterprise/3x1/vm_connectivity_domain.svg)
+![DC Connectivity Domain](/img/concepts/enterprise/3x1/dc_connectivity_domain.svg)
+
+Each workload has a single option (possibly with nerd knobs) of what connectivity domain to be connected to, and only
+workloads in a given runtime domain could be part of its connectivity domain.
+
+In short: Connectivity Domains are strongly coupled to Runtime Domains.
+
+A central tenant of [Cloud Native](https://github.com/cncf/toc/blob/main/DEFINITION.md) is Loose Coupling.
+
+What Runtime Domain a workload is running in is a [non-sequitur](https://en.wikipedia.org/wiki/Non_sequitur_(literary_device)) to its communications needs.  Workloads that are part of
+the same App need Connectivity between each other no matter where they are running.
+
+One example of this problem is connectivity between workloads running in multiple K8s Clusters in a 
+multi-cloud/hybrid cloud environment:
+
+![K8s MultiCluster](/img/concepts/enterprise/k8s_multi_cluster.svg)
+
+How do workloads communicate independent of where they are running?
+
+It's not just a problem of cluster to cluster communication.  In the diagram below: 
+- The Red Pods need to communicate with each other
+- The Greed Pods need to communicate with each other
+- The Red Pod with the Green outline needs to communicate with both the other Red Pods *and* the other Green Pods
+- Pods in different clusters that are neither Red nor Green should be communicating with each other.
+
```

Comment:
Note/discussion to redo pieces with a legend to help color-blindness people.

Reply:
Colors added.